server/controllers/cv-controller.js

```diff
@@ -0,0 +1,14 @@
+const githubClient = require('../lib/github-client.js');
+
+module.exports.getData = (username, res) => {
+  const userProfilePromise = githubClient.getUserProfile(username);
+  const userPullRequestsPromise = githubClient.getUserPullRequests(username);
+  Promise.all([userProfilePromise, userPullRequestsPromise])
+  .then(([userProfile, userPullRequests]) => res.render('cv', { userProfile: userProfile, userPullRequests: userPullRequests }))
```

Comment:
👍 Really nice Array destructuring.

You could use a neat syntax tidy up this line a little bit
```js
res.render('cv', { userProfile: userProfile, userPullRequests: userPullRequests })
```
could become 
```js
res.render('cv', { userProfile, userPullRequests })
```
If you're putting a variable in an object and you want the key to be the same name as the variable, you don't have to explicitly do `variableName: variableName`. A quick example would be

```js
const name = 'Simon';
const age = 24;
const person = { name, age }
person // {name: 'Simon', age: 24}
```